### DISCUSSION TITLE
安装过程中出现错误。分发名称: 'Ubuntu 20.04 LTS' 错误代码: 0x80240438

### DISCUSSION THREAD

**Discussion by pyxiaoxiaobo** [community/community#144679] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/d26e66b7-74f0-499d-a3a5-b6cc52d88863)

**Comment by shinybrightstar** [comment]
Hi @pyxiaoxiaobo , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
